packages/react-data-grid/src/__tests__/CellExpand.spec.js

```diff
@@ -1,68 +1,60 @@
 import React from 'react';
+import { mount } from 'enzyme';
+
 import CellExpander from '../CellExpander';
 import { CellExpand } from 'common/constants';
-import { mount } from 'enzyme';
 
-describe('CellExpand', () => {
-  let testElement;
-  let getFakeProps = (expanded) => {
-    let expandableOptions = {expanded};
-    let onCellExpand = jasmine.createSpy();
-    return {
-      expandableOptions,
-      onCellExpand
-    };
+const setup = (overrideExpandableOptions = {}) => {
+  const props = {
+    expandableOptions: {
+      expanded: true,
+      ...overrideExpandableOptions
+    },
+    onCellExpand: jasmine.createSpy()
   };
 
-  const renderComponent = (props) => {
-    const wrapper = mount(<CellExpander {...props} />);
-    return wrapper;
+  return {
+    props,
+    wrapper: mount(<CellExpander {...props} />)
   };
+};
 
+describe('CellExpand', () => {
   it('should create an instance of CellExpand', () => {
-    let fakeProps = getFakeProps(false);
-    testElement = renderComponent(fakeProps);
-    expect(testElement.find(CellExpander).length).toBe(1);
+    const { wrapper } = setup({ expanded: false });
+
+    expect(wrapper.find('.rdg-cell-expand').length).toBe(1);
   });
 
   it('should render correctly when expanded is true', () => {
-    let fakeProps = getFakeProps(true);
-    testElement = renderComponent(fakeProps);
-    expect(testElement.state('expanded')).toBeTruthy();
-    expect(testElement.find('span.rdg-cell-expand').text()).toBe(CellExpand.DOWN_TRIANGLE);
+    const { wrapper } = setup({ expanded: true });
+
+    expect(wrapper.state('expanded')).toBeTruthy();
```

Comment:
Rather than setting the state, can we use `simulate` users behavior

```
wrapper.simulate('click'); // Not sure if this will work but the idea is to simulate a click event
expect(wrapper.find('span').text()).toBe(CellExpand.DOWN_TRIANGLE);
```